src/main/java/io/r2dbc/h2/util/Check.java

```diff
@@ -0,0 +1,25 @@
+package io.r2dbc.h2.util;
+
+/**
+ * Utilities for inspecting some conditions.
+ */
+public final class Check {
```

Comment:
Let's keep the `isPresent` check within `DefaultCodecs`. Also, we should allow passing a `ClassLoader`.

Reply:
Summarizing required changes:
* Remove `Check` class.
* Move `findClass` method into `DefaultCodecs` clas.
* Rename `findClass` method into `isPresent`.
* Add `ClassLoader` as parameter to `isPresent`.

Ok?